docs/csharp/language-reference/builtin-types/integral-numeric-types.md

```diff
@@ -94,7 +94,7 @@ The type of an integer literal is determined by its suffix as follows:
 
 If the value represented by an integer literal exceeds <xref:System.UInt64.MaxValue?displayProperty=nameWithType>, a compiler error [CS1021](../../misc/cs1021.md) occurs.
 
-The value represented by an integer literal can be implicitly converted to a type with a smaller range than the determined type of the literal. That's possible when the value is within the range of the destination type:
+If the determined type of an integer literal is `int`, the value represented by the literal can be implicitly converted to `sbyte`, `byte`, `short`, `ushort`, `uint`, or `ulong`. That's possible when the value is within the range of the destination type:
```

Comment:
@BillWagner I was wrong with generalization in my previous PR on this article: `byte c = 4;` compiles, but `byte c = 4U;` — not. Fixing that now.

Reply:
Hmm. I thought I had tested that. I verified that this update is correct.